pkg/gui/controllers/branches_controller.go

```diff
@@ -459,6 +462,9 @@ func (self *BranchesController) createPullRequestMenu(selectedBranch *models.Bra
 			{
 				LabelColumns: fromToLabelColumns(branch.Name, self.c.Tr.DefaultBranch),
 				OnPress: func() error {
+					if len(branch.UpstreamBranch) == 0 {
```

Comment:
I'd just call `handleCreatePullRequest` from here (We don't want to do `self.createPullRequest(branch.Name, "")` below and looks like we're doing the right thing on line 212